### PR TITLE
Replace image scale computing algorithm by GD library.

### PR DESCRIPTION
**Pull request by Stanin84**

### CODE REVIEW

File: tests/eyes.images.php.tests/TestImageUtils.php [hunk 1]
```diff
@@ -28,5 +28,7 @@ public function testResize()
         $t = gettimeofday();
         $timestamp = date("Y_m_d H_i_s", $t['sec']);
         imagepng($resizedImage, "minions-400x250_$timestamp.png");
+
+        $this->assertTrue(true);
```
Comment (Trawais) [inline tests/eyes.images.php.tests/TestImageUtils.php:33]: Add comment, why this must be here.

File: tests/tutorials/AppiumNativeTest.php [hunk 2]
```diff
@@ -57,4 +63,7 @@ public function demo()
 }
 
 $test = new HelloWorld();
-$test->demo();
\ No newline at end of file
+try {
```
Comment (Trawais) [inline tests/tutorials/AppiumNativeTest.php:66]: why you've added the catching the exception?

File: tests/tutorials/SeleniumTest.php [hunk 3]
```diff
@@ -61,4 +67,7 @@ public function demo()
 }
 
 $test = new HelloWorld();
-$test->demo();
\ No newline at end of file
+try {
```
Comment (Trawais) [inline tests/tutorials/SeleniumTest.php:70]: Same here

File: tests/tutorials/SeleniumTest.php [hunk 4]
```diff
@@ -61,4 +67,7 @@ public function demo()
...
+    $test->demo();
+} catch (Exception $e) {
+}
```
Comment (Trawais) [inline tests/tutorials/SeleniumTest.php]: Please, always put an trailing newline character at the end of each file.

**Review by Trawais — CHANGES_REQUESTED** [review]

File: tests/tutorials/AppiumNativeTest.php (hunk 2 above)
Comment (Stanin84) [reply tests/tutorials/AppiumNativeTest.php:66]: Because of Code Analysis warning.

File: tests/tutorials/SeleniumTest.php (hunk 3 above)
Comment (Stanin84) [reply tests/tutorials/SeleniumTest.php:70]: Because of Code Analysis warning.

**Review by Trawais — APPROVED** [review]

**Pull request merged**